BaseFilter/sections/specific.txt

```diff
@@ -17385,6 +17385,7 @@ fijivillage.com#?#.contentleft > p > small:contains(/^ADVERTISEMENT$/)
 biertijd.com,biertijd.xxx#?##wrapper > div#sidebarcontainer > h2:contains(/Advertenties|Sponsors|Aanrader/)
 mumbaimirror.indiatimes.com#?##content > div[class*=" "]:has(> div > div > a[onclick])
 mumbaimirror.indiatimes.com#?#.latestCmmtData > .mrtoday > div[class*=" "]:has(> div > div > a[onclick])
+circleofcricket.com##div.container-full.add:has(> a[href]:first-child)
```

Comment:
Maybe it can be simplified to:
```adblock
circleofcricket.com##.add
```
By the way, perhaps this rule:
```adblock
||circleofcricket.com/assets/adv/
```
could be also added.
If I'm not wrong this path is used only for ads on this website.

Reply:
.add style has only padding, I think it can possibly be used for other elements, so I added some other criteria (full width container, has link as first child)

Reply:
I just suppose that in this case `add` is a misswrite for `ad` and it's used only for advertisement.
But if you think that it might cause incorrect blocking, then the current rule is fine.